src/Apps/Components/AppShell.tsx

```diff
@@ -26,6 +26,15 @@ export const AppShell = props => {
     }
   }, [routeConfig])
 
+  /**
+   * Let our end-to-end tests know that the app is hydrated and ready to go
+   */
+  useEffect(() => {
+    if (typeof window !== "undefined") {
```

Comment:
Inside of `useEffect` there will always be a window

Reply:
Oh, right. Thanks!